src/MarkdownEditor.js

```diff
@@ -65,7 +78,7 @@ var MarkdownEditor = React.createClass({
     };
 
     if(this.props.styles.styleMarkdownEditorHeader) {
-      styleMarkdownEditorHeader = this.props.styles.styleMarkdownEditorHeader;
+      Object.assign(styleMarkdownEditorHeader, this.props.styles.styleMarkdownEditorHeader);
```

Comment:
yep, you're right, I forgot about that. I'm kind of green when it comes to javascript -- I'm a backend dev that's been living in Java for a long time. I'll look into that.